api/internal/plugins/fnplugin/fnplugin.go

```diff
@@ -72,12 +72,13 @@ func toStorageMounts(mounts []string) []runtimeutil.StorageMount {
 func NewFnPlugin(o *types.FnPluginLoadingOptions) *FnPlugin {
 	return &FnPlugin{
 		runFns: runfn.RunFns{
-			Functions:      []*yaml.RNode{},
-			Network:        o.Network,
-			EnableStarlark: o.EnableStar,
-			EnableExec:     o.EnableExec,
-			StorageMounts:  toStorageMounts(o.Mounts),
-			Env:            o.Env,
+			Functions:             []*yaml.RNode{},
+			Network:               o.Network,
+			EnableStarlark:        o.EnableStar,
+			EnableExec:            o.EnableExec,
+			StorageMounts:         toStorageMounts(o.Mounts),
+			Env:                   o.Env,
+			ContinueOnEmptyResult: true,
```

Comment:
Should we make this option configurable?

And it's better to put it into struct `FnPluginLoadingOptions`

Reply:
Done